src/transformers/models/pvt/configuration_pvt.py

```diff
@@ -49,7 +49,7 @@ class PvtConfig(PretrainedConfig):
             The input image size
         num_channels (`int`, *optional*, defaults to 3):
             The number of input channels.
-        num_encoder_blocks (`[int]`, *optional*., defaults to 4):
+        num_encoder_blocks (`[int]`, *optional*, defaults to 4):
```

Comment:
```suggestion
        num_encoder_blocks (`int`, *optional*, defaults to 4):
```
something seems a bit wrong with this one